cvat-core/src/api-implementation.ts

```diff
@@ -390,19 +405,56 @@ export default function implementAPI(cvat) {
             };
         }
 
-        const reportsData = await serverProxy.analytics.quality.conflicts(updatedParams);
+        const conflictsData = await serverProxy.analytics.quality.conflicts(updatedParams);
+        const conflicts = conflictsData.map((conflict) => new QualityConflict({ ...conflict }));
+        conflicts.count = conflictsData.count;
+        return conflicts;
+    };
 
-        return reportsData.map((conflict) => new QualityConflict({ ...conflict }));
+    cvat.analytics.quality.settings.get.implementation = async (filter: any) => {
+        interface FilterParams {
+            id?: number, taskId?: number, projectId?: number
+        }
+        const { id, taskId, projectId }: FilterParams = filter;
+
+        const settings = await serverProxy.analytics.quality.settings.get(id, taskId, projectId);
+        if (settings) {
+            return new QualitySettings({ ...settings });
+        }
+        return null;
     };
 
-    cvat.analytics.quality.settings.get.implementation = async (taskID: number) => {
-        const settings = await serverProxy.analytics.quality.settings.get(taskID);
-        return new QualitySettings({ ...settings });
+    cvat.analytics.quality.settings.update.implementation = async (settingsId: number, values: any) => {
+        const settings = await serverProxy.analytics.quality.settings.update(settingsId, values);
+
+        if (settings) {
+            return new QualitySettings({ ...settings });
+        }
+        return null;
     };
 
-    cvat.frames.getMeta.implementation = async (type, id) => {
-        const result = await serverProxy.frames.getMeta(type, id);
-        return new FramesMetaData({ ...result });
+    cvat.analytics.quality.settings.create.implementation = async (values: any) => {
+        const settings = await serverProxy.analytics.quality.settings.create(values);
+
+        if (settings) {
+            return new QualitySettings({ ...settings });
+        }
+        return null;
+    };
+
+    cvat.analytics.quality.settings.defaults.implementation = async () => {
+        const scheme = await cvat.scheme.get();
```

Comment:
From the inside of cvat-core, better to call implementation method directly.
I do not think such call should trigger plugins if there are any

So, just ``await cvat.scheme.get.implementation()`` or ``serverProxy.scheme.get()``.

Reply:
Updated